owncloudApp/src/main/res/layout/remove_files_dialog.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center">
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/cardView"
```

Comment:
An ID like this is not suitable, it is very general and could collide easily with other `CardView`s in the app, in addition that we don't know which `CardView` in the app we're referring to 😄. I think we don't even use this ID, we can safely delete it 👍

Reply:
In this case, I've deleted that attribute because it could be confusing with others card views.